vignettes/articles/InputFiles.Rmd

```diff
@@ -18,35 +18,47 @@ run_name=ssp245
 Here, comments start with a semicolon (`;`); *section names* (`core`) are enclosed in square brackets; and a *property* (`run_name`) is assigned a *value* (`ssp245`). Properties are guaranteed to be unique within a particular section, and all sections match one-to-one with a model [component](ComponentAPI.html).
 
 
-**Table of model parameters**
```

Comment:
Please update [SSP245 file](/inst/input/hector_ssp245.ini) to [SSP245 file](https://github.com/JGCRI/hector/tree/master/inst/input/hector_ssp245.ini), note that this path will not work until the hector v3 branch is merged into master